### DISCUSSION TITLE
Is it supported by Z2MQTT as device ?

### DISCUSSION THREAD

**Discussion by kevincaradant** [Koenkk/zigbee2mqtt#8000] [Devices]
Hi guys, 

I'm wondering if this device is currently supported by the Z2M ? I'mnot sure if the brand is moes or tuya ...

I can see a label "2MQTT" but not sure it's for Zigbee2MQTT :)

I don't know what is the reference of this remote but to be honest it's the only one I got like that with "real" button and not touch button. 

So do you think it will work ? 

![image](https://user-images.githubusercontent.com/6093175/125132056-0f6f4000-e104-11eb-8252-9462038a0003.png)

Thank you in advance

**Comment by kevincaradant** [comment]
Hi

I finally found the device, it's this one

https://zigbee.blakadder.com/Moes_ZS-SR4-2169.html

But it's seems that it's not supported yet by Z2MQTT :)

**Reply by nickrout** [reply]
Yes but now you have a perfect chance to make it work, and contribute to the community :)

Although it does seem supported. Search TS0044 on here https://www.zigbee2mqtt.io/information/supported_devices.html

**Reply by kevincaradant** [reply]
I don't know if I can do it without a sniffer ect.
I'm not comfortable with it yet.
I need to check how to help to support it. I just see some converters and need to find the good one. In my case I know its related to tuya but I didn't yet purchase it because I was not sure that I could do work well with Z2MQTT but now you said yes, I will purchase some remotes :). 

However, why did you link to me the TS0044, its a device with 4 buttons, but it's not the same device between a remote and a switch? Sorry if I didn't understand your answer with it :)

Thanks! ') 


Edit: oh OK my bad I just understood about the reference which is noted on the link that I given... My bad :)

**Reply by nickrout** [reply]
Seems to me it is just a different set of buttons on the same chipset and should work.

**Reply by kevincaradant** [reply]
OK. 

I ordered them few days ago. I will tell you if it works as expected when I will receive them. :)

Thanks

**Reply by kevincaradant** [reply]
Hello

I received them and i can tell you that it works :D

However I don't know why but I got "null" as value for the battery level, so something wrong with that but otherwise everything works as expected.

MQTT publish: topic 'zigbee2mqtt/remote_bathroom', payload '{"action":"","battery":null,"linkquality":69}'
